src/content-tags-drawer/ContentTagsDrawer.jsx

```diff
@@ -20,9 +21,14 @@ import Loading from '../generic/Loading';
 /** @typedef {import("../taxonomy/data/types.mjs").TaxonomyData} TaxonomyData */
 /** @typedef {import("./data/types.mjs").Tag} ContentTagData */
 
-const ContentTagsDrawer = () => {
+const ContentTagsDrawer = ({ id, onClose }) => {
   const intl = useIntl();
-  const { contentId } = /** @type {{contentId: string}} */(useParams());
+  const params = useParams();
+  let contentId = id;
+
+  if (contentId === undefined) {
+    contentId = params.contentId;
+  }
```

Comment:
I didn't like this, but I don't know a better way to do it and maintain compatibility with the iframe in the studio. Could you comment on why we use the component props and the query params?

Reply:
I think this is fine for now, but I agree we should add a comment saying that once the legacy iframe is no longer used, we can remove `useParams()`

Reply:
Added [here](https://github.com/openedx/frontend-app-course-authoring/pull/852/files/92eea935910c56f2237d6ccb35c093426bf5b420..845e5ed08d60f44278014332151e9e8f4d1fd2d3)